datalake/src/main/java/com/sequenceiq/datalake/flow/chain/DatalakeUpgradeFlowEventChainFactory.java

```diff
@@ -29,7 +31,7 @@ public FlowTriggerEventQueue createFlowTriggerEventQueue(DatalakeUpgradeFlowChai
         chain.add(new DatalakeTriggerBackupEvent(DATALAKE_TRIGGER_BACKUP_EVENT.event(),
                 event.getResourceId(), event.getUserId(), event.getBackupLocation(), "",
                 event.getSkipOptions(),
-                DatalakeBackupFailureReason.BACKUP_ON_UPGRADE, event.accepted()));
+                DatalakeBackupFailureReason.BACKUP_ON_UPGRADE, List.of(DatabaseType.HIVE.toString().toLowerCase()), event.accepted()));
```

Comment:
Just curious, any reason we are choosing to only backup Hive for upgrades? I may have missed this convo.

Reply:
During the upgrade, It doesn't need to ensure that it's backup since it will continue to actually live in Hive. hence the lack of the need to back it up